README.md

```diff
@@ -201,11 +208,24 @@ GPU, much faster than previous methods, while the generated instances can be dir
 - [fast: C++/CUDA](https://github.com/MrNeRF/gaussian-splatting-cuda)
 - [nerfstudio: python/CUDA](https://github.com/nerfstudio-project/gsplat)
 
-### Viewers 
+### Viewers
 - [WebGL Viewer 1](https://github.com/antimatter15/splat)
 - [WebGL Viewer 2](https://github.com/cvlab-epfl/gaussian-splatting-web)
 - [Three.js](https://github.com/mkkellogg/GaussianSplats3D)
 - [A-Frame](https://github.com/quadjr/aframe-gaussian-splatting)
+- [Nerfstudio Unofficial](https://github.com/yzslab/nerfstudio/tree/gaussian_splatting)
+- [Nerfstudio Viser](https://github.com/nerfstudio-project/viser)
+- [Blender (Editor)](https://github.com/ReshotAI/gaussian-splatting-blender-addon/tree/master)
+- [WebRTC viewer](https://github.com/dylanebert/gaussian-viewer)
+- [iOS & Metal viewer](https://github.com/laanlabs/metal-splats)
+
+## Paid Implementations
+
+### Game Engines 
+- [Unreal Plugin](https://vrlab.akiya-souken.co.jp/en/products/threedgaussianplugin/)
+
+### Viewers
+- [Tellusim WebGPU](https://tellusim.com/07_hello_splatting/)
```

Comment:
The code is built on top of the Tellusim SDK what I believe is a paid service, again this could stay in the Paid / Closed Source  category as reference

Reply:
Right, I did not look to closely. So I would rather consider to remove them.

Why not list instead some good tweets? I mean, there are some that explain lots of what is happening behind the curtains of this great technology as for instance: https://x.com/the_ross_man/status/1704331583786778895?s=20